Makefile

```diff
@@ -1,5 +1,4 @@
 SHELL := /bin/bash
-PORT := 7001
```

Comment:
We still have `${PORT}` in `run-with-docker`, so it might be nicer to keep this defined at the top to save having to specify it. But not sure anyone uses `run-with-docker` anyway, so up to you.

Reply:
fixed, thanks. I've rebased this change in